clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -752,6 +752,30 @@ static bool isDeclaredInSYCLNamespace(const Decl *D) {
   return ND && ND->getName() == "sycl";
 }
 
+static bool isSYCLPrivateMemoryVar(VarDecl *VD) {
```

Comment:
These two functions have been moved from down below to make them visible at their new call sites. The argument type for addScopeAttrToLocalVars has also been changed.

Thanks